vector/src/main/java/im/vector/app/features/widgets/permissions/RoomWidgetPermissionBottomSheet.kt

```diff
@@ -102,8 +102,9 @@ class RoomWidgetPermissionBottomSheet :
     }
 
     override fun onCancel(dialog: DialogInterface) {
+        viewModel.handle(RoomWidgetPermissionActions.BlockWidget)
```

Comment:
Maybe should we reuse the existing `doDecline` method?

Reply:
https://github.com/vector-im/element-android/pull/5187#discussion_r813882777 😄

Reply:
I see. It is up to you but we could create a sub method without including call to `dismiss()` as you suggested?